compiler/test/runnable/variadic.d

```diff
@@ -1118,6 +1118,49 @@ void test15417()
 }
 
 
+/***************************************/
+// https://issues.dlang.org/show_bug.cgi?21425
+
+import core.stdc.stdarg;
+import core.stdc.stdio;
+
+extern(C) void f5(int dummy, ...)
+{
+    va_list ap;
+
+    va_start(ap, dummy);
+    int x = va_arg!int(ap);
+    assert(x == 5);
+    va_end(ap);
+
+    va_start(ap, dummy);
+    int y = va_arg!int(ap);
+    assert(y == 5);
+    va_end(ap);
+}
+
+void test21425()
+{
+    f5(0, 5);
+}
+
+/*********************************************/
+// https://issues.dlang.org/show_bug.cgi?23409
+
+void printf10(const(char)* fmt, ...){
+    for(int i = 0; i < 10; i++){
+        va_list args;
+        va_start(args, fmt);
+        vprintf(fmt, args);
```

Comment:
The output of this function is not tested - can you add a [`RUN_OUTPUT`}(https://github.com/dlang/dmd/tree/master/compiler/test#test-configuration) section to the start of this file?

Reply:
It's not really necessary. The nature of the bug is it will crash.

Reply:
replaced with vsprintf